### PR TITLE
refLength was not always updating

### PR DESCRIPTION
**Pull request by martinRenou**
The refLength update wasn't always done. This had the impact of eventually breaking the clipplane effect which depends on it.

### CODE REVIEW

**Review by trungleduc — APPROVED** [review]

**Comment by trungleduc** [comment]
Thanks @martinRenou

File: packages/base/src/3dview/mainview.tsx [hunk 1]
```diff
@@ -796,17 +794,17 @@ export class MainView extends React.Component<IProps, IStates> {
...
+
+      // Update clip plane size
+      this._clippingPlaneMeshControl.geometry = new THREE.PlaneGeometry(
+        this._refLength * 10,
+        this._refLength * 10
+      );
+    } else {
+      this._refLength = null;
     }
+    console.log('reflength is', this._refLength);
```
Comment (martinRenou) [inline packages/base/src/3dview/mainview.tsx:807]:
```suggestion
```

**Comment by martinRenou** [comment]
Bot please update snapshots

**Comment by martinRenou** [comment]
Some visual changes are not ok, putting as draft

**Comment by martinRenou** [comment]
Bot please update snapshots

**Comment by SylvainCorlay** [comment]
Could this be the reason why the pointer "ball" was sometimes oversized / undersized?

**Comment by martinRenou** [comment]
> Could this be the reason why the pointer "ball" was sometimes oversized / undersized?

Very likely yes. The pointer ball diameter should now stay roughly a tenth of the reference length.

**Pull request merged**